CONTRIBUTING.md

```diff
@@ -19,6 +19,7 @@ Please understand that we only bundle languages with the monaco editor that have
 - create `$/src/basic-languages/{myLang}/{myLang}.ts`
 - create `$/src/basic-languages/{myLang}/{myLang}.test.ts`
 - edit `$/src/basic-languages/monaco.contribution.ts` and register your new language
+- create `$/website/index/samples/sample.{myLang}.txt`
```

Comment:
Can you create a separate PR for this?
It makes it much easier to understand what is going on / to revert functionality in case of emergencies.

Reply:
Sure, I created #3210. I could revert them here, but I think the simplest way forward is to merge the main branch again after #3210 has been merged.